### DISCUSSION TITLE
Unable to get result, when using expand option

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3936] [Q&A] [answered]
for this url: http://192.168.55.102:8090/api/collections/matches/records?perPage=30&expand=series_id

i am getting below results, not getting expand info

<img width="488" alt="Screenshot 2023-12-13 at 4 57 43 AM" src="https://github.com/pocketbase/pocketbase/assets/94731739/16ef1c05-1544-41ca-bc12-4246fbfa47a2">

in schema i am using relation column only
<img width="677" alt="Screenshot 2023-12-13 at 4 59 05 AM" src="https://github.com/pocketbase/pocketbase/assets/94731739/b29239a5-3cfb-49cf-9c73-fc3b985183a7">

**Comment by valiantlynx** [comment]
expand is part of filter.
so not `http://192.168.55.102:8090/api/collections/matches/records?perPage=30&expand=series_id`
instead something like `http://192.168.55.102:8090/api/collections/matches/records?perPage=30&filter=&expand=series_id%2Cteam1_id%2Cteam2_id`

**Reply by ganigeorgiev** [reply]
This is incorrect. `expand` is its own query parameter.

**Comment by nikhil-varma-mudunuri** [comment]
even after trying this link, unable to get expand results

http://192.168.55.102:8090/api/collections/matches/records?perPage=30&filter=&expand=series_id

<img width="568" alt="Screenshot 2023-12-13 at 11 34 14 AM" src="https://github.com/pocketbase/pocketbase/assets/94731739/6fe74918-6424-480a-a2b5-e5dbb42aec01">

**Reply by ganigeorgiev** [reply, marked as answer]
Double check the View API rule of the `series` collection. Also make sure that there is no cache involved.

If it still doesn't work, please export your collections configuration as json (Admin UI > Settings > Export collections) and I'll try to test it.

**Comment by nikhil-varma-mudunuri** [comment]
it worked when i change the API rules for expand tables.
